apps/remix-ide/src/app/tabs/test-tab.js

```diff
@@ -103,7 +103,7 @@ module.exports = class TestTab extends ViewPlugin {
 
     if (eChecked) {
       checkAll.checked = true
-      if ((this.readyTestsNumber === this.runningTestsNumber || this.hasBeenStopped) && document.getElementById('runTestsTabStopAction').innerText === 'Stop') {
+      if ((this.readyTestsNumber === this.runningTestsNumber || this.hasBeenStopped) && document.getElementById('runTestsTabStopAction').innerText === ' Stop') {
```

Comment:
Instead of adding a space to the compared value, we can trim trailing spaces.
```javascript
const runBtnInnerText = document.getElementById('runTestsTabStopAction').innerText

if ((this.readyTestsNumber === this.runningTestsNumber || this.hasBeenStopped) && runBtnInnerText.trim() === 'Stop')
```
This will prevent ide from breaking when an extra space is added or removed in the future.

Reply:
yes, seems reasonable